lib/models/key.coffee

```diff
@@ -33,19 +33,13 @@ auth = require('../auth')
 #		console.log(keys)
 ###
 exports.getAll = (callback) ->
-
-	auth.getUserId (error, id) ->
-		return callback(error) if error?
-
+	auth.getUserId().then (id) ->
 		return pine.get
 			resource: 'user__has__public_key'
 			options:
 				filter:
 					user: { id }
-		.tap (keys) ->
-			if _.isEmpty(keys)
```

Comment:
Yeah, I'm returning an empty array instead of throwing an error in `getAll` functions.